easy-pbkdf2.js

```diff
@@ -72,7 +72,7 @@ EasyPbkdf2.prototype = {
 	 * @returns {SlowBuffer} (optional)
 	 */
 	"random": function( bytes, callback ) {
-		if ( _.isFunction( callback ) ) {
+		if ( callback instanceof Function ) {
```

Comment:
I'd like to avoid using `instanceof` checks because they'll fail for values from other realms/contexts (like node's `vm`). No idea why someone would be passing in functions from other contexts in here, but it could happen.